app/src/main/java/chat/rocket/android/helper/ServerPolicyHelper.java

```diff
@@ -71,6 +71,14 @@ private static String removeProtocol(String hostname) {
     return hostname.replace("http://", "").replace("https://", "");
   }
 
+  private static String removeTrailingSlash(String hostname) {
+    if (hostname.charAt(hostname.length() - 1) != '/') {
```

Comment:
The intent (improving performance) is a little hard to guess without any comments, because`removeTrailingSlash` is called not so frequently enough to consider performance improvement.

Please add comment for `if (hostname.charAt(hostname.length() - 1) != '/')`.